src/main/java/com/softech/coursesvc/service/UserService.java

```diff
@@ -0,0 +1,36 @@
+package com.softech.coursesvc.service;
+
+import com.softech.coursesvc.model.Course;
+import com.softech.coursesvc.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.web.bind.annotation.GetMapping;
+
+import java.util.List;
+@Service
+public class UserService {
+    @Autowired
+    public UserRepository userRepository;
+    public List<Course> getAllUser() {
+        return userRepository.findAll();
+
+    }
+
```

Comment:
All these classes are duplicate...you need to pull from master and then add you code on top of it.. please remove all duplicate classes